### ISSUE TITLE
Add support for dynamic locales

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/energy-skate-park#350]
Discussed on slack with @arouinfar that this is probably not needed for the upcoming publication. But it may be quick to add before we start a dev test this week. Ill give it a try but bail if it takes more than an hour today.

**Comment by jessegreenberg** [comment]
I converted everything to string Properties but there is going to be a lot more to be a lot more work required to support dynamic layout. I can't take this on at the moment so Ill stop working on this until it is requested.

**Comment by jessegreenberg** [comment]
During status meeting today it was confirmed that publication for region/culture does not need to include dynamic locales.

**Comment by samreid** [comment]
Closing as duplicate of https://github.com/phetsims/energy-skate-park/issues/397
